tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = {py35,py36}-master
+envlist = py{35,36}-django111, py{35,36}-django{20,21,22}
```

Comment:
why not simply do `py{35,36}-django{111,20,21,22}`

Reply:
I was facing some issues in the format to decide so I followed the format of some other repo in edx. Changed the `envlist` as per suggestion.